std-bits/base/src/main/java/org/enso/base/Time_Utils.java

```diff
@@ -79,29 +90,6 @@ public static LocalDate date_adjust(LocalDate date, AdjustOp op, Period duration
     }
   }
 
-  public static ZonedDateTime datetime_adjust(
-      ZonedDateTime datetime, AdjustOp op, Period period, Duration duration) {
```

Comment:
Surprising. How do you handle `Period`? It is not Truffle interop type and needs to be still treated via host interop, I think.